cmdstanpy/lib.py

```diff
@@ -429,7 +429,7 @@ def summary(self) -> pd.DataFrame:
             suffix='.csv', prefix=tmp_csv_file, dir=TMPDIR, text=True
         )
         cmd = '{} --csv_file={} {}'.format(
-            cmd_path, tmp_csv_path, ' '.join(self.csv_files)
+            cmd_path, tmp_csv_path.replace("\\", "/"), ' '.join(self.csv_files).replace("\\", "/")
```

Comment:
CmdStan's tools should be able to handle Windows paths on a Windows system - why is this necessary?